src/components/Account/Account.tsx

```diff
@@ -225,7 +235,7 @@ const Account = ({ panelClassName, panelHeaderClassName }: Props): JSX.Element =
         formSection({
           label: t('account.terms_and_tracking'),
           saveButton: t('account.update_consents'),
-          onSubmit: (values) => updateConsents(formatConsentsFromValues(publisherConsents, values)),
+          onSubmit: (values) => updateConsents(formatConsentsFromValues(publisherConsents, values)) as Promise<never>,
```

Comment:
Same effect, without casting
```suggestion
          onSubmit: async (values) => {
            await updateConsents(formatConsentsFromValues(publisherConsents, values))
          },
```